apps/native/app/src/screens/settings/settings.tsx

```diff
@@ -171,6 +174,7 @@ export const SettingsScreen: NavigationFunctionComponent = ({
         title={intl.formatMessage({ id: 'setting.screenTitle' })}
         onClosePress={() => Navigation.dismissModal(componentId)}
         style={{ marginHorizontal: 16 }}
+        loading={userProfile.loading}
```

Comment:
~Same here. `userProfile.loading && userProfile.data`?

Reply:
In this case this is true